airbyte-cdk/python/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 0.16.3
```

Comment:
nit: but given this is additive non-breaking functionality (and not a bug fix), we should bump the minor version so `0.17.0`